content/java/concepts/strings/terms/valueOf/valueOf.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.valueOf()'
+Description: 'Returns the string representation of given value.'
```

Comment:
```suggestion
'Returns the string representation of a given value.'
```

Reply:
Hey @Dusch4593, Do I have to remove the Description tag too?